src/themes/types.js

```diff
@@ -668,6 +668,7 @@ export type AnimationT = {
   easeInQuintic: string,
```

Comment:
For consistency, should we add "Curve" at the end of the easeXXQuintic or remove it from the other ones? (cc @sandgraham)

Reply:
Yes. I'll update that.